pages/article/[pid].tsx

```diff
@@ -220,9 +219,9 @@ const ArticlePage = (initialArticle) => {
 
             <Col flex = '24%'>
                   <UserArticle bio = {article.author.bio ? article.author.bio: 'I am a senior frontend developer, passionate about creative coding and building interactive prototypes mixing science, art & technology. I also spend time mentoring, contribution to OSS and speaking.'}
```

Comment:
If there is not bio, occupation, location, or joined data for the author, we should just display nothing but the title of the fields.